### PR TITLE
[PLDM] Resolved static analysis errors in PLDM

### PR DESCRIPTION
**Pull request by ArchanaKakani**
Resolved nullptr dereference errors.

Signed-off-by: ArchanaKakani <archana.kakani@ibm.com>

### CODE REVIEW

File: libpldm/pdr.c [hunk 1]
```diff
@@ -184,8 +188,8 @@ uint32_t pldm_pdr_add_hotplug_record(pldm_pdr *repo, const uint8_t *data,
 
 	pldm_pdr_record *record = make_new_record(
 	    repo, data, size, record_handle, is_remote, terminus_handle);
-	add_hotplug_record(repo, record, prev_record_handle);
-	return record->record_handle;
+	return (add_hotplug_record(repo, record, prev_record_handle));
+	// return record->record_handle;
```
Comment (manojkiraneda) [inline libpldm/pdr.c:192]: remove the comment

File: libpldm/pdr.c [hunk 2]
```diff
@@ -199,9 +203,10 @@ uint32_t pldm_pdr_add_after_prev_record(pldm_pdr *repo, const uint8_t *data,
 
 	pldm_pdr_record *record = make_new_record(
 	    repo, data, size, record_handle, is_remote, terminus_handle);
-	add_record_after_record_handle(repo, record, prev_record_handle);
+	return (
+	    add_record_after_record_handle(repo, record, prev_record_handle));
 
-	return record->record_handle;
+	// return record->record_handle;
```
Comment (manojkiraneda) [inline libpldm/pdr.c:209]: remove comment.

File: libpldm/pdr.c (hunk 1 above)
Comment (ArchanaKakani) [reply libpldm/pdr.c:192]: Done

File: libpldm/pdr.c (hunk 2 above)
Comment (ArchanaKakani) [reply libpldm/pdr.c:209]: Done

**Pull request closed without merging**